licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/assessment/ProposedAddressRejectedSpec.groovy

```diff
@@ -0,0 +1,77 @@
+package uk.gov.justice.digital.hmpps.licences.specs.assessment
+
+import geb.spock.GebReportingSpec
+import spock.lang.Shared
+import spock.lang.Stepwise
+import uk.gov.justice.digital.hmpps.licences.pages.assessment.ProposedAddressRejectedPage
+import uk.gov.justice.digital.hmpps.licences.pages.assessment.ProposedAddressCurfewAddressPage
+import uk.gov.justice.digital.hmpps.licences.pages.assessment.ProposedAddressCurfewAddressChoicePage
+
+import uk.gov.justice.digital.hmpps.licences.util.Actions
+import uk.gov.justice.digital.hmpps.licences.util.TestData
+
+@Stepwise
+class ProposedAddressRejectedSpec extends GebReportingSpec {
+
+    @Shared
+    TestData testData = new TestData()
+
+    @Shared
+    Actions actions = new Actions()
+
+    def setupSpec() {
+      testData.loadLicence('assessment/unstarted')
+      actions.logIn('CA')
+    }
+
+    def cleanupSpec() {
+      actions.logOut()
+    }
+
+    def "Click 'save and continue' without selecting a radio button"() {
+
+        given: 'I am on the ProposedAddressRejectedPage'
+        to ProposedAddressRejectedPage, testData.markAndrewsBookingId
+
+        when: 'I select the continue button'
+        find('#continueBtn').click()
```

Comment:
These test look good.

Is there any reason why this button can't be on the ProposedAddressRejectedButton?

Generally you should be interacting via the page object rather than doing bespoke calls to target elements directly.

Reply:
agreed. Done

Reply:
Seems like you've updated other places just not this one?

Reply:
oops! 
done